src/components/section-list.component.js

```diff
@@ -15,9 +15,9 @@ type Props = {
   noItemsMessage: string,
   buttonAction: Function,
 };
-
+const hairlineWidth = StyleSheet.hairlineWidth;
 const Section = styled.View`
-  margin-top: 15;
+  /* margin-top: 15; */
```

Comment:
Do you want to remove this line?

Reply:
yes, i think some style need to change, so i change something , but it's no complete

Reply:
Sure, friendly reminding, please keep a PR independent. And you can test different kinds of styles locally or in other branches. Then our maintainers will not be notified to review. :)